xain_fl/coordinator/coordinator.py

```diff
@@ -428,15 +448,36 @@ def _handle_end_training_round(
             if self.current_round >= self.num_rounds - 1:
                 logger.info("Last round over", round=self.current_round)
                 self.state = State.FINISHED
+                self._write_metrics_fail_silently({"state": self.state})
             else:
                 self.current_round += 1
                 self.epoch_base += self.epochs
+                self._write_metrics_fail_silently({"round": self.current_round})
                 # reinitialize the round
                 self.select_participant_ids_and_init_round()
 
         logger.debug("Send EndTrainingRoundResponse", participant_id=participant_id)
         return EndTrainingRoundResponse()
 
+    def _write_metrics_fail_silently(
```

Comment:
minor point: do we (or will we) also have a "`_write_metrics_fail_loudly`"? Because `write_metrics` seems good enough for me.

Reply:
probably not 😁 
I couldn't find a better name to explain that for now we ignore any errors that are raised from the metric store until we have agreed on how to handle exceptions.

Reply:
perhaps a comment to that effect in the function would be enough then, but I'll leave it up to you!

Reply:
I have added a comment and linked the ticket.